Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:alpine
+
+WORKDIR /opt/git-badges
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
+
+CMD ["npm", "run", "start"]
```

Comment:
can be written as `CMD ["npm", "start"]`

Reply:
Yeah, I just used 'run' because you used it in your documentation too.